web/modules/custom/eulogin/src/Routing/JoinupEuLoginRouteSubscriber.php

```diff
@@ -29,6 +29,12 @@ protected function alterRoutes(RouteCollection $collection): void {
         ->setDefaults(['_controller' => UserRegisterRedirectController::class . '::redirectUserRegister'])
         ->setRequirements(['_user_is_logged_in' => 'FALSE']);
     }
+
+    // Always deny access to '/user/login'. Users are expected to log in through
+    // EU Login.
+    if ($route = $collection->get('user.login')) {
+      $route->setRequirement('_access', 'FALSE');
+    }
```

Comment:
This is not correct. It was decided in the standup that we don't deny the access to the route but we disable (or hide) the form.

Reply:
This is temporary and is going to be removed in scope of ISAICP-5760. I will add a `@todo` here.